integration-test/src/test/java/org/apache/iotdb/db/it/IoTDBConfigNodeIT.java

```diff
@@ -204,6 +228,597 @@ public void removeAndStopConfigNodeTest() {
+              "",
+              new HashSet<>(),
+              new ArrayList<>());
+      authorizerResp = client.queryPermission(authorizerReq);
+      status = authorizerResp.getStatus();
+      Assert.assertEquals(TSStatusCode.SUCCESS_STATUS.getStatusCode(), status.getCode());
+      Assert.assertEquals(
+          privilege, authorizerResp.getAuthorizerInfo().get(IoTDBConstant.COLUMN_PRIVILEGE));
+
+      // list all role of user
+      authorizerReq =
+          new TAuthorizerReq(
+              AuthorOperator.AuthorType.LIST_ROLE.ordinal(),
+              "tempuser0",
+              "",
+              "",
+              "",
+              new HashSet<>(),
+              new ArrayList<>());
+      authorizerResp = client.queryPermission(authorizerReq);
+      status = authorizerResp.getStatus();
+      assertEquals(TSStatusCode.SUCCESS_STATUS.getStatusCode(), status.getCode());
+      roleList.remove("temprole1");
+      assertEquals(roleList, authorizerResp.getAuthorizerInfo().get(IoTDBConstant.COLUMN_ROLE));
+
+      // list all user of role
+      authorizerReq =
+          new TAuthorizerReq(
+              AuthorOperator.AuthorType.LIST_USER.ordinal(),
+              "",
+              "temprole0",
+              "",
+              "",
+              new HashSet<>(),
+              new ArrayList<>());
+      authorizerResp = client.queryPermission(authorizerReq);
+      status = authorizerResp.getStatus();
+      assertEquals(TSStatusCode.SUCCESS_STATUS.getStatusCode(), status.getCode());
+      userList.remove("tempuser1");
+      userList.remove("root");
+      assertEquals(userList, authorizerResp.getAuthorizerInfo().get(IoTDBConstant.COLUMN_USER));
+
+      // revoke role from user
+      authorizerReq =
+          new TAuthorizerReq(
+              AuthorOperator.AuthorType.REVOKE_ROLE_FROM_USER.ordinal(),
+              "tempuser0",
+              "temprole0",
+              "",
+              "",
+              new HashSet<>(),
+              new ArrayList<>());
+      status = client.operatePermission(authorizerReq);
+      assertEquals(TSStatusCode.SUCCESS_STATUS.getStatusCode(), status.getCode());
+
+      // list root privileges
+      authorizerReq =
+          new TAuthorizerReq(
+              AuthorOperator.AuthorType.LIST_USER_PRIVILEGE.ordinal(),
+              "root",
+              "",
+              "",
+              "",
+              new HashSet<>(),
+              new ArrayList<>());
+      authorizerResp = client.queryPermission(authorizerReq);
+      status = authorizerResp.getStatus();
+      assertEquals(TSStatusCode.SUCCESS_STATUS.getStatusCode(), status.getCode());
+      for (int i = 0; i < PrivilegeType.values().length; i++) {
+        assertEquals(
+            PrivilegeType.values()[i].toString(),
+            authorizerResp.getAuthorizerInfo().get(IoTDBConstant.COLUMN_PRIVILEGE).get(i));
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
+      fail(e.getMessage());
```

Comment:
Just throw any Exceptions in the interface, we don't need to catch and print them since it's a test code